content/documentation/pages/1-installation/1-local/3-manual.md

```diff
@@ -0,0 +1,104 @@
+---
+path: 'installation/local/manual'
+title: 'Manual'
+description: 'Manual installation'
+---
+
+# Manual Installation
+
+If Docker does not suit your needs, you can manually install the parts you need to run Spring Cloud Data Flow.
+
+## Download server jars
+
+1.  Download the Spring Cloud Data Flow Server by using the following
+    command:
+
+        wget https://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-server/{project-version}/spring-cloud-dataflow-server-{project-version}.jar
```

Comment:
Can we use variables here so that the user can just do a copy paste?

Reply:
Changed reference of `{project-version}` to `%dataflow-version%`.   Added a placeholder variable for `version-type-lowercase`